app/src/styles/App.scss

```diff
@@ -296,6 +317,9 @@ $hunit: '%';
                 margin-bottom: 0;
                 border-left: none;
                 border-right: none;
+                #full-screen-btn {
+                    top: -30px;
+                }
```

Comment:
Where is this styling coming from?

Reply:
I have removed the Detail tag wrapper as it no longer made semantical sense. But to make the "view full screen" button stay where it is, i needed to add some style to it